psmdb/percona-server-for-mongodb-4.0-template.yml

```diff
@@ -64,12 +64,12 @@
         set -x
 
         if [ -f /usr/bin/apt-get ]; then
-            sudo apt-get -y install wget
+            sudo DEBIAN_FRONTEND=noninteractive apt-get -y install wget
         else
             sudo yum -y install wget redhat-lsb-core
         fi
 
-        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/${PSM_BRANCH}/percona-packaging/scripts/psmdb_builder.sh
+        wget https://raw.githubusercontent.com/percona/percona-server-mongodb/v4.0/percona-packaging/scripts/psmdb_builder.sh
```

Comment:
no. branch should be a param

Reply:
yep, Evgeniy is right, branch should be param